l1-verifier/scripts/remote.ts

```diff
@@ -0,0 +1,18 @@
+import hre from "hardhat";
```

Comment:
What is this script for?

Reply:
It's mentioned here https://github.com/ensdomains/evmgateway/pull/4/files#diff-a1176ce8a8e70ac565ce2aa2e27bc770a2ac7b36e803e985bf2dd9419b634c64R34